extensions/amp-story/0.1/amp-story-bookend.js

```diff
@@ -369,12 +373,20 @@ export class Bookend {
           if (!response) {
             return null;
           }
-
-          this.config_ = {
-            shareProviders: response['share-providers'],
-            relatedArticles:
-                relatedArticlesFromJson(response['related-articles']),
-          };
+          // TODO(#14591): Clean when old version is deprecated.
+          if (response[BOOKEND_VERSION] === BOOKEND_VERSION_2) {
+            this.config_ = {
+              [BOOKEND_VERSION]: BOOKEND_VERSION_2,
+              components: BookendComponent
+                  .buildFromJson(response['components']),
+            };
+          } else {
+            this.config_ = {
```

Comment:
Maybe `console.warn('Version 1 of the amp-story bookend is deprecated.  Use version 2: [link]')`

As a side note, we should publish docs somewhere so that `[link]` can link to something 😄   That part can come in a separate PR.

Reply:
Ok, will do this in a follow-up PR after this 👍